sycl/test-e2e/Matrix/joint_matrix_unaligned_k_impl.hpp

```diff
@@ -0,0 +1,154 @@
+#include <iostream>
```

Comment:
Code of the second .hpp file is a complete duplication of joint_matrix_out_bounds_impl.hpp. The only difference is MATRIX_K parameter. we can have single .hpp file and separate .cpp files defining MATRIX_K to avoid duplication.